README.md

```diff
@@ -73,6 +73,16 @@ roles:
 
 Follow the example above to get started. if the `htpasswd` key is removed from the environment file, the protection will be removed on the next deploy.
 
+Adding / Removing Basic Authentication
+-------------------------------
```

Comment:
Can you make it so the dashes match the length of the title to stay consistent with the other titles?